client/src/pages/checkout/confirmation/confirmation.styles.js

```diff
@@ -0,0 +1,142 @@
+import styled from 'styled-components';
+
+export const ConfirmationBanner = styled.img`
+  width: 100%;
+  height: auto;
+  margin-top: -80px;
```

Comment:
When I ran the app, this banner image covered the navigation bar, so I wasn't able to see the nav section on the checkout confirmation page. I would suggest erasing this line to move this banner image down to make the navigation section visible.